src/renderer/src/helpers/UtilsHelper.ts

```diff
@@ -57,4 +57,24 @@ export abstract class UtilsHelper {
       return aValue < bValue ? 1 : -1
     })
   }
+
+  static copyToClipboard(text: string) {
+    navigator.clipboard.writeText(text)
+  }
+
+  static uuid() {
+    let dt = new Date().getTime()
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
+      const r = (dt + Math.random() * 16) % 16 | 0
+      dt = Math.floor(dt / 16)
+      return (c === 'x' ? r : (r & 0x3) | 0x8).toString(16)
+    })
+  }
```

Comment:
You don't need this. You can use the "uuid" lib.
```suggestion

import * as uuid from 'uuid'
export class UtilsHelper {
   ...methods

  static uuid() {
    return uuid.v4()
  }
}
```